espeak_phonemizer/__init__.py

```diff
@@ -190,29 +190,25 @@ def _phonemize_mem_stream(
         finally:
             self.libc.fclose(phonemes_file)
 
-    def _phonemize_no_stream(
-        self, text: str, phoneme_separator: Optional[str]
-    ) -> List[str]:
+    def _phonemize_no_stream(self, text: str, \
+        phoneme_separator: [Optional[str])](url) -> List[str]:
```

Comment:
You've added an additional `)` here that you're not actually opening.

Reply:
@PureTryOut @synesthesiam

Thankfully Michael cleaned up that typo in a later commit.

[Here](https://github.com/rhasspy/espeak-phonemizer/commit/0ac9d44b9f5ba9c016230e4c32fcf7f1f4a56217)

Reply:
Don't worry, I saw